### PR TITLE
Small hotfix for cass-management call and fix e2e-testing

### PR DESCRIPTION
**Pull request by burmanm**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -75,7 +64,58 @@ jobs:
...
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Export reaper-operator image
+        run: |
+          docker save k8ssandra/reaper-operator:latest --output=/tmp/k8ssandra-reaper-operator.tar
+      - name: Upload reaper-operator image
+        uses: actions/upload-artifact@v2
+        with:
+          name: reaper-operator
+          path: /tmp/k8ssandra-reaper-operator.tar
+  run_e2e_test:
```
Comment (jsanda) [inline .github/workflows/build.yml:80]: Are we pushing the image to docker hub before running the e2e tests? If so would it better to push only after the tests pass?

File: pkg/config/config.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ func ReplicationToString(r api.ReplicationConfig) string {
 }
 
 func ReplicationToConfig(dcName string, r api.ReplicationConfig) []map[string]string {
-	replicationConfig := make([]map[string]string, 1)
+	replicationConfig := make([]map[string]string, 0, 1)
```
Comment (jsanda) [inline pkg/config/config.go:31]: I cannot recall why this was `make([]map[string]string, 1)` vs `make([]map[string]string, 0)`. Out of curiosity why the change to `make([]map[string]string, 0, 1)`?

**Review by jsanda — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (burmanm) [reply .github/workflows/build.yml:80]: We're not pushing in PRs at all.

File: pkg/config/config.go (hunk 2 above)
Comment (burmanm) [reply pkg/config/config.go:31]: Because []map[string]string, 1 leaves the first item as null since we use append. With 0, 1 the append will write to position 0 (using the preallocated position).

**Review by jsanda — APPROVED** [review]
